src/utils/numbers.ts

```diff
@@ -0,0 +1,26 @@
+export interface FactorMap {
+    divisor: number,
+    labels: string[],
+}
+
+export const bytesMap: FactorMap = {
+    divisor: 1024,
+    labels: [' Bytes', ' KiB', ' MiB', ' GiB', ' TiB', ' PiB', ' EiB', ' ZiB', ' YiB']
+}
+
+export const kiloMap: FactorMap = {
+    divisor: 1000,
+    labels: ['', 'k', 'M']
```

Comment:
I cannot help but notice that these are not as many as in the `bytesMap`. Should they not?
Furthermore: Should this array not also extend in the opposite direction: "milli", "nano", ("pico", "femto", but maybe really not necessary)?

Reply:
Please see changes in README.md for the exact meaning. This map name is just an example and is the mapping function is currently not tested to support values < 1.

Reply:
Err... did you forget to push them? Two Readmes have changed:

- In the root-level Readme only a link to jest is added,
- in the utils-level Readme this is the entirety of changes (after pruning PR 13 from it):
```
## numbers

Number handling and formatting.

```typescript
import {bytesMap, getFactor, kiloMap} from "./numbers";
const posts = 1300;
const kilo = getFactor(posts, kiloMap);
const bytes = getFactor(posts, bytesMap);
console.log(`Your wrote ${(posts / kilo.factor).toFixed(1)} ${kilo.label} posts with a size of ${(posts / bytes.factor).toFixed(0)} ${bytes.label} `);
// You wrote 1.3k posts with a size of 1 KiB
```
You also wrote:
> This PR is not about formatting numbers. Please consider reading `README.md`.

but the Readme right there in the headline tells me, it's _all_ about "Number handling and formatting." so...

Reply:
I corrected my statement about formatting: This PR supports features not covered by `Intl.NumberFormat`.

Reply:
Okay, works for me.